_posts/2022-05-27-pro-components-with-faasd.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "Exploring OpenFaaS Pro without Kubernetes"
```

Comment:
What other titles could you come up with?

Reply:
The Event-driven Edge with OpenFaaS

Reply:
Handle Events at the Edge with OpenFaaS